### ISSUE TITLE
Doesn't work when reddit preferences is set to use old reddit on standard reddit URLs

### ISSUE THREAD

**Issue by brian6932** [pauldraper/unreddit#8]
![image](https://user-images.githubusercontent.com/18603393/98735347-39077a00-2371-11eb-8b8a-6a153b6abae1.png)
It works on new reddit UI when "Use new Reddit as my default experience" is checked, but it doesn't work on standard URLs when it's unticked. I've been trying to figure out why the extension was working for me in incognito mode, I understand why now lol.

**Comment by Brandonmccoub2** [comment]
Same issue here, did you ever find a way to fix it other than setting New Reddit as the default?

**Comment by brian6932** [comment]
Nah, but this kinda works: https://greasyfork.org/en/scripts/407466
